Classes/APIBots/utils.js

```diff
@@ -98,11 +98,31 @@ function isWallet(msg) {
 }
 
 function isVerticalName(msg) {
-  return msg === 'curated' || msg === 'factory' || msg === 'playground'
+  return (
+    msg === 'curated' ||
+    msg === 'presents' ||
+    msg === 'collaborations' ||
+    msg === 'collabs' ||
+    msg === 'heritage' ||
+    msg === 'factory' ||
+    msg === 'playground' ||
+    msg === 'explorations' ||
+    msg === 'engine' ||
+    msg.startsWith('curatedseries')
```

Comment:
```suggestion
    msg.startsWith('curated series')
```

I think there should be a space?

Reply:
nope, since artbot has already deburred and de-whitespaced the message by this point